public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -107,6 +113,70 @@ function Onboarding(svl, audioEffect, compass, form, handAnimation, mapService,
         onboardingModel.triggerStartOnboarding();
     };
 
+    /**
+     * Fetches the string data of the PointCloud object for the tutorial. If Google removes the tutorial's
+     * panos, the PointCloud data is stored in a .txt file.
+     */
+    function fetchTutorialPointCloud() {
+        var client = new XMLHttpRequest();
+        client.open('GET', svl.rootDirectory + "doc/TutorialPointCloud.dat");
+        client.onreadystatechange = function() {
+            tutorialPC = client.responseText;
+        };
+        client.send();
+    }
+
+    /**
+     * @returns {PointCloud} - returns the PointCloud object for the tutorial panorama
+     */
+    function getTutorialPointCloud() {
+        if (typeof tutorialPC === "string") {
+            tutorialPC = JSON.parse(tutorialPC);
+        }
+        return tutorialPC;
+    }
+
+    /**
+     * Sets the mini map to be transparent for everything except for yellow pin man.
```

Comment:
lol I don't know about the gender of the yellow pin, I'd just go with "yellow pin" :)